### DISCUSSION TITLE
Need some help to understand some config parameters

### DISCUSSION THREAD

**Discussion by milan-patartics** [swar/Swar-Chia-Plot-Manager#472] [Q&A]
I would like to create a job that starts 4 plotting threads with 30 minutes offset for each. But at the same time I also want to early start new plot creations when a thread's progress reaches phase 4 (+10 minutes).

I am bit confused by the max_concurrent and max_concurrent_with_start_early options.
I think the following settings for the scenario I described would work, but I would like to ask you for your opinion to validate that it will do what I am expecting.

max_concurrent: 4
max_concurrent_with_start_early: 8
stagger_minutes: 30
max_for_phase_1: 4 (?)
concurrency_start_early_phase: 4
concurrency_start_early_phase_delay: 10

So would this config start 4 threads at T, T+30, T+60, T+90 and keep creating new plot processes every time a thread reaches phase 4 plus 10 minutes?

Thank you!

**Comment by Cminer3207** [comment]
If you find the answer... let us know. 
This is one of the questions we all have. 
Btw, excellent description of how that process works. 
I’m assuming you want to go +10 minutes to avoid the brief surge in CPU and Temp space that happens at the beginning of Phase 4?

**Reply by milan-patartics** [reply]
Hopefully @swar will spend a few minutes on us to explain it.

**Reply by realdevnullius** [reply]
Well, seen it's being ignored I'm gonna assume this is exactly how it works :) I like the explanation, will put it to the test. So much testing :(
